packages/react-components/react-link/src/components/Link/useLinkStyles.ts

```diff
@@ -10,20 +10,14 @@ export const linkClassNames: SlotClassNames<LinkSlots> = {
 
 const useStyles = makeStyles({
   focusIndicator: createCustomFocusIndicatorStyle({
-    borderBottomColor: 'transparent',
     textDecorationColor: tokens.colorStrokeFocus2,
     textDecorationLine: 'underline',
     textDecorationStyle: 'double',
   }),
   // Common styles.
   root: {
     backgroundColor: 'transparent',
-    borderTopStyle: 'none',
-    borderLeftStyle: 'none',
-    borderRightStyle: 'none',
-    borderBottomColor: 'transparent',
-    borderBottomStyle: 'solid',
-    borderBottomWidth: tokens.strokeWidthThin,
+    ...shorthands.borderStyle('none'),
```

Comment:
I think this can be removed entirely? The less styles are used, less classnames are generated by griffel; less classname on element reduces style calculation time. Also border style generates 4 classes.

Reply:
Removed!